### ISSUE TITLE
Installation on Ubuntu not working

### ISSUE THREAD

**Issue by aocenas** [gitify-app/gitify#401]
When trying to run the AppImage which is the default download for linux on the webpage I get:
```
./Gitify-3.0.4.AppImage 
(electron) The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397
Checking for update
Update for version 3.0.4 is not available (latest version: 3.0.4, downgrade is disallowed).
checkForUpdatesAndNotify called, downloadPromise is null
```
and it just hangs there not doing anything.

If I install through the deb package from releases it looks like it is installed (it is shown in Ubuntu Software installed list) but running the app does nothing.

**Comment by rauldipeas** [comment]
Same here.
```
(electron) The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397
APPIMAGE env is not defined, current application is not an AppImage
```

**Comment by brpaz** [comment]
Please check if the tray icon is not hidden by right-clicking in the system tray in the menubar,

I had the some issue on Fedora 32 with Gnome 3.36 and managed to found it.

![icon](https://imgur.com/bsOe4du)

I guess the icon is not loading or is black

**Comment by rauldipeas** [comment]
No icon is showing on system tray, i'm using _systray_ on XFCE.
https://docs.xfce.org/xfce/xfce4-panel/systray

**Comment by xcq1** [comment]
Same issue on Ubuntu Budgie.

**Comment by Komposten** [comment]
I'm also using XFCE with systray and I have the same issue. Gitify appears to start properly but does not show up in the tray so I can't interact with it.

**Comment by rauldipeas** [comment]
Latest version is working well with XFCE systray, but i had to manually change color of the icon.
![image](https://user-images.githubusercontent.com/12108031/85854765-66b9a100-b78b-11ea-9097-cd9f93f4dca0.png)

**Comment by Komposten** [comment]
Just confirmed that 3.1.0 works for me as well (Manjaro with XFCE systray). The window appears at the top of the screen rather than the bottom (where the systray is), but that's just a minor annoyance.

**Comment by rauldipeas** [comment]
Same issue here, with the position of the window.

**Comment by Adi-df** [comment]
I've got the same probleme on Ubuntu 18.04

**Comment by eggplants** [comment]
I've got the same probleme on Ubuntu 20.04 LTS

**Comment by MostHated** [comment]
I am having this issue on Pop_os 20.10.

I see the following message:
```shell
~/.files » ./Gitify-4.2.0.AppImage                             mosthated@pop-os
Checking for update
Update for version 4.2.0 is not available (latest version: 4.2.0, downgrade is disallowed).
checkForUpdatesAndNotify called, downloadPromise is null
```
No window comes up, nothing on taskbar, etc. I see it running under the system processes, though.

<details open>
<summary>Screenshot</summary>

![](https://i.imgur.com/0TSW0TF.png)

</details>

**Comment by bmulholland** [comment]
We're starting to maintain this again, and have released a new version. Could anyone confirm if this issue is still happening with v4.4.0?

**Comment by bmulholland** [comment]
Sounds like it may be working, so closing this out... but let me know if it's still happening and we can re-open.

**Comment by Grtschnk** [comment]
Unfortunately, issue as described by https://github.com/gitify-app/gitify/issues/401#issuecomment-796282369 still exists for me on Arch when trying to run 4.5.1.

I played around with the source on an earlier version, as far as I could tell the Electron tray minimisation is broken on Gnome/Linux. When I tweaked the app to start in focus and/or without tray I could get it to somewhat run.

I remember seeing some PR/Issue regarding upgrading Electron, has this gone through or still waiting to happen? (IIRC it was quite a version bump and therefore a bit tricky)

OS: Arch Linux x86_64 
Host: Latitude 7420 
Kernel: 6.6.4-arch1-1 
Uptime: 23 hours, 57 mins 
Packages: 1785 (pacman), 31 (flatpak) 
Shell: zsh 5.9 
Resolution: 1920x1080 
DE: GNOME 45.2 
WM: Mutter 
WM Theme: Adwaita 
Theme: Adwaita-dark [GTK2/3] 
Icons: Adwaita [GTK2/3] 
Terminal: gnome-terminal 
CPU: 11th Gen Intel i7-1185G7 (8) @ 4. 
GPU: Intel TigerLake-LP GT2 [Iris Xe G 
Memory: 24174MiB / 31827MiB
